package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "diaspora",
+  "version": "0.7.17.0",
+  "repository": "git@github.com:diaspora/diaspora.git",
+  "author": "diaspora* team",
+  "license": "AGPL-3.0",
+  "dependencies": {
+    "jquery.textchange": "gist:424774",
```

Comment:
This gist seems to be the first original source of this jquery extension

Reply:
I don't like adding a dependency from gist in production. I don't really understand what happened to this as there was something with a version number before (so a release?) and now there is only a gist left?

But also, I don't think we need this dependency anymore at all. The only place that I can find where this is used is in `charcount.js` which is the counter for when posting to services with limited length (like twitter for example). And we can probably just replace it with the native `input` event. And getting rid of dependencies is always something I like, especially dependencies that were only used at one place and can be replaced with very little code.